types/types.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+// const (
+// 	AppName = "CRUD"
+// )
+
+const (
+	// Bech32PrefixAccAddr defines the Bech32 prefix of an account's address
```

Comment:
This is a bit worrying, has this been tested in a zone?

Reply:
I'll try this out tomorrow.

Reply:
I tried it on a one-node zone on my machine with CRUD and sending tokens.
Works fine.

Obviously we do not have integration tests to validate nothing else is
broken.